frontend/src/pages/ExploreData/ExploreDataPage.tsx

```diff
@@ -76,6 +76,7 @@ function ExploreDataPage() {
     <div id="ExploreData" className={styles.ExploreData}>
       <div className={styles.CarouselContainer}>
         <Carousel
+          aria-label="Mad lib selection carousel"
```

Comment:
I do not think you need an aria-label for the carousel container. I think just the back/forward buttons should have aria-labels, since they are interactive.